pixloc/utils/data.py

```diff
@@ -21,6 +21,7 @@ class Paths:
     results: Optional[Path] = None
     global_descriptors: Optional[Path] = None
     hloc_logs: Optional[Path] = None
+    log_path: Optional[Path] = None
```

Comment:
The path entry is actually `hloc_logs` and already exists. I'd rather use this name instead.